### DISCUSSION TITLE
Is there a way to have the chart scroll along the x or y axis

### DISCUSSION THREAD

**Discussion by eamodio** [naver/billboard.js#2570] [Q&A]
I am rendering a timeseries chart with a lot of data and I would like to be able to scroll horizontally through time. And additionally I also have horizontal row/swim-lane data on the y-axis, and would like to be able to scroll vertically as well.

Is that possible?

**Comment by netil** [comment]
well, there's no clear way to do so.
There're 2 possible ways you can try, but I'm not sure it will meet your needs.

One simple way is to make chart holder element's size smaller and specify greater size option value.
<img src=https://user-images.githubusercontent.com/2178435/158595845-c4b84682-27e2-48b3-84e8-833f8f9b3303.gif width=200>
- Demo: https://jsfiddle.net/netil/2pq80mx1/9

Second way is during the initialization zoom-in to specific range, and after by dragging chart shape area can navigate back and forth.
<img src=https://user-images.githubusercontent.com/2178435/158595865-411d4c14-e8b7-49c3-b72c-2c1d57c53573.gif width=380>
- Demo: https://jsfiddle.net/netil/2pq80mx1/11/

Recommended way to navigate long ranged period day, is by providing a [subchart](https://naver.github.io/billboard.js/demo/#Interaction.SubChart) navigation.

Hope this helps.

**Reply by eamodio** [reply]
@netil Thanks! I'll take a look at the zoom option -- that might be workable. That will only work for the X axis though right?
